ui/app/components/pgp-file.js

```diff
@@ -64,6 +64,9 @@ export default Component.extend({
   ),
 
   actions: {
+    handleToggle(e) {
```

Comment:
I opted to add this action instead of glimmerize this file to keep scope small and because this file should eventually just be replaced with the [Hds::Form::FileInput::Field](https://helios.hashicorp.design/components/form/file-input), instead of continuing to maintain inflexible components